test/rulesets/LinearAlgebra/structured.jl

```diff
@@ -161,4 +161,9 @@
             end
         end
     end
+
+    @testset "Tridiagonal" begin
+        res, pb = rrule(Tridiagonal, [1, 4], [2, 3, 4], [5, 3])
```

Comment:
Can this be written usually ChainRulesTestUtils's test rule
I think it should be just 
```suggestion
        test_rrule(Tridiagonal, [1.0, 4.0], [2.0, 3.0, 4.0], [5.0, 3.0])
```